### ISSUE TITLE
Male voice

### ISSUE THREAD

**Issue by Zaverichintanoxolo** [shivammehta007/Neural-HMM#2]
Hey,
Thanks for sharing the code. Is there any example for male voice synthesis? I want to train on custom male voice dataset .

**Comment by shivammehta007** [comment]
Hello! 
I have mainly trained with the datasets containing female voices, different accents but the voices were coincidently female. As much as I have seen, you should not face any trouble with the male voice dataset as the model trains relatively faster, you will see the learning and synthesising within 1500 iterations (depending on the size of the dataset) and can determine if you need to tweak some hyperparameters.

Also, a tip will be to look at the tensorboard while training, it can give very insightful information, including the synthesised mel-spec at specific intervals (can be changed in `hparams.py`), forward alignments, loss curves, change in parameter's distributions, gradient norms, etc.

Otherwise, If you have any specific corpus in mind we can look into it as well :)

**Comment by chikiuso** [comment]
I am also looking for a male voice, hope to have a male model soon, thanks for your help.

**Comment by shivammehta007** [comment]
Hello, 

I have updated the repository with a [male voice](https://github.com/shivammehta007/Neural-HMM/releases/download/Neural-HMM/Neural-HMM-Male.ckpt) based on [RyanSpeech](https://arxiv.org/abs/2106.08468) corpus . 

Also, we have noticed it is easier to finetune this checkpoint on a male voice compared to the LJSpeech based checkpoint which is easier to finetune for a female voice.
